dotnet/src/SemanticKernel.Core/Memory/SemanticTextMemory.cs

```diff
@@ -33,6 +35,7 @@ public sealed class SemanticTextMemory : ISemanticTextMemory
     }
 
     /// <inheritdoc/>
+    [Experimental("SKEXP0003")]
```

Comment:
Marking the type as experimental should result in everything on it being considered experimental as well. You shouldn't need to mark the individual methods when you've also annotated the type.

Reply:
Yeah... I think I went overboard on that one. Should be fixed now. I did, however, need to add it to any of the static methods.